dali/aux/optical_flow/optical_flow_adapter.h

```diff
@@ -16,14 +16,23 @@
 #define DALI_AUX_OPTICAL_FLOW_OPTICAL_FLOW_ADAPTER_H_
 
 #include <dali/pipeline/data/backend.h>
-#include "dali/kernels/tensor_view.h"
+#include <dali/kernels/backend_tags.h>
+#include <dali/kernels/tensor_view.h>
 
 namespace dali {
 namespace optical_flow {
 
 namespace detail {
 
-using Backend = kernels::StorageGPU;
+template<typename ComputeBackend>
+struct Compute2Storage {
+  using type = kernels::StorageCPU;
+};
+
+template<>
+struct Compute2Storage<kernels::ComputeGPU> {
```

Comment:
```suggestion
struct compute_to_storage_backend<kernels::ComputeGPU> {
```
Traits are traditionally written with lowercase snake case.

Reply:
Done